docs/_src/api/openapi/generate_openapi_specs.py

```diff
@@ -12,11 +13,19 @@
 
 os.environ["PIPELINE_YAML_PATH"] = pipeline_path
 
-from rest_api.application import get_openapi_specs
+from rest_api.application import get_openapi_specs, haystack_version
 
 # Generate the openapi specs
 specs = get_openapi_specs()
 
 # Dump the specs into a JSON file
-with open(f"openapi.json", "w") as f:
+with open("openapi.json", "w") as f:
     json.dump(specs, f, indent=4)
+
+# Remove rc versions of the specs from the folder
+for specs_file in os.listdir():
+    if os.path.isfile(specs_file) and "rc" in specs_file and specs_file.endswith(".json"):
```

Comment:
We could also use the Path library here and use `.suffix` instead of `endswith`. In many other parts of Haystack, we are using Path lib instead of os but I don't see a disadvantage of os here so there is no reason to change it.

Reply:
Totally right! I'll change this.